apps/web-giddh/src/assets/css/global/vat-report.scss

```diff
@@ -1,7 +1,45 @@
 .vat-report {
     .vat-report-table {
+        &.mat-mdc-table {
+            &.zimbabwe-vat-report-table {
+                tr {
+                    height: 44px !important;
+
+                    th {
+                        padding: 5px 10px !important;
+                    }
+
+                    th,td {
+                        border:  1px solid var(--color-table-primary-content-bar-border) !important;
+        
+                        &[colspan="2"]{
+                            text-align: center;
+                        }
+                        &.d-none {
```

Comment:
why add display: none;  css 
when already using bootstrap class d-none

Reply:
to over ride mat table display property, without using more descriptive information I unable to apply display none property